Xamarin.PropertyEditing.Mac/PropertyEditorPanel.cs

```diff
@@ -129,11 +115,12 @@ void Initialize ()
 				RefusesFirstResponder = true,
 				AutoresizingMask = NSViewResizingMask.WidthSizable,
 				SelectionHighlightStyle = NSTableViewSelectionHighlightStyle.None,
+				RowHeight = 24,
 			};
 
-			// create columns for the panel
-			NSTableColumn propertiesList = new NSTableColumn ("PropertiesList") { Title = PropertyListTitle };
-			NSTableColumn propertyEditors = new NSTableColumn ("PropertyEditors") { Title = PropertyEditorTitle };
+			// TODO: localize
+			NSTableColumn propertiesList = new NSTableColumn (PropertyListColId) { Title = "Properties" };
```

Comment:
I really think this should be singular. The column name refers to a single Property.